chia/full_node/full_node.py

```diff
@@ -2259,10 +2259,10 @@ async def respond_transaction(
                 if self.mempool_manager.get_spendbundle(spend_name) is not None:
                     self.mempool_manager.remove_seen(spend_name)
                     return MempoolInclusionStatus.SUCCESS, None
-                assert self.mempool_manager.peak
-                cost, status, error = await self.mempool_manager.add_spend_bundle(
-                    transaction, cost_result, spend_name, self.mempool_manager.peak.height
-                )
+                if self.mempool_manager.peak is not None:
+                    cost, status, error = await self.mempool_manager.add_spend_bundle(
+                        transaction, cost_result, spend_name, self.mempool_manager.peak.height
+                    )
             if status == MempoolInclusionStatus.SUCCESS:
```

Comment:
it looks like, with this change, the next error will be that `status` is not defined.